docs/setup/settings.asciidoc

```diff
@@ -63,3 +63,67 @@ information and all requests.
 The minimum value is 100.
 `status.allowAnonymous`:: *Default: false* If authentication is enabled, setting this to `true` allows
 unauthenticated users to access the Kibana server status API and status page.
+
```

Comment:
Sounds good. I just need to figure out how to create a chunk, since otherwise it's a choice between a huge blob in the install page, or a huge blob in the settings page.